### ISSUE TITLE
[Feature request] File created/last-modified timestamp

### ISSUE THREAD

**Issue by rafalchmielarski-nes** [SendSafely/Java-Client-API#13]
Hello,

As part of the use case I am currently working on, I would like to be able to download from SendSafely workspace only latest files (ones that have been created/modified in last 24 hours period).

At the moment the SendSafely Java Client doesn't seem to allow listing files based on datetime filter or access the created/last-modfied timestamp information on a specific file, so this needs to be handled separately.

Ideally, having getter(s) on a File object which allow to access created and/or last-modified timestamp (probably last-modified can capture both cases) would make that possible, so for example:

```
Package pkg = sendsafely.getPackageInformationFromLink(myLink);
long someTimeInThePast = 1589192861000L; 

// Download files newer than given timestamp
for(File file : pkg.getFiles()) {
    long fileLastModifiedTimestamp = file.getLastModifiedTimestamp(); // epoch timestamp, can also be a Date type etc.
    if(fileLastModifiedTimestamp > someTimeInThePast) {
        java.io.File newFile = sendsafely.downloadFile(pkg.getPackageId(), file.getFileId(),   pkg.getKeyCode());
        System.out.println("Downloaded File with name: " + newFile.getName() + " Length: " + newFile.length());
    }
}
```
I was wondering if it would be possible to request similar changes and whether they can be added to next/future release?

Thanks
Rafal Chmielarski

**Comment by SendSafely-GitHub** [comment]
Hi Rafal --

Thank you for your feedback. We plan on addressing this as part of our next release. There will be a property on each file you can read that shows the date/time that the file was uploaded. We will reply back here once the code has been released.

Thanks!

**Comment by SendSafely-GitHub** [comment]
This feature was implemented in our v3.1.6 release: https://github.com/SendSafely/Java-Client-API/releases/tag/v3.1.6
